pandas/core/arrays/base.py

```diff
@@ -361,7 +361,7 @@ def isna(self) -> ArrayLike:
         """
         raise AbstractMethodError(self)
 
-    def _values_for_argsort(self) -> np.ndarray:
+    def _values_for_argsort(self) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
This is a breaking change for ExtensionArray authors. I am not sure how many already rely on this, but should we deprecate it first? (eg handle both cases (only values and values+mask) where it is used, and raise warning if only values are returned)

Reply:
Or actually, is it needed to change this? As we can get the mask just from `self.isna()` ?